library/templates/dcp_mappluto_historical.yml

```diff
@@ -0,0 +1,54 @@
+dataset:
+  name: &name dcp_mappluto
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    url:
+      path: s3://edm-publishing/mappluto/mappluto_{{ version }}.zip
```

Comment:
This link is invalid

Reply:
The last version I got to work was 20v7 which is the last one updated version in edm-publishing

Reply:
Interesting. Not really sure what the "historical" mappluto is and why this action moves it from edm publishing to data library. @td928 do you have any insights? Otherwise this question may have to wait for Amanda

Reply:
yeah or maybe Lynn probably knows something about this as well